package.json

```
@@ -1,23 +1,25 @@
 {
   "private": true,
-  "workspaces": [
-    "packages/*"
-  ],
   "scripts": {
-    "build": "turbo run build",
-    "dev": "turbo run dev --parallel",
     "lint": "turbo run lint",
     "test": "turbo run test",
-    "clean": "turbo run clean && rm -rf node_modules",
-    "format": "prettier --write \"**/*.{js,md}\""
+    "prettier": "prettier -w .",
+    "prepare": "husky install",
+    "eslint": "cross-env TIMING=1 eslint --max-warnings 0 --config .eslintrc.js"
   },
   "devDependencies": {
-    "prettier": "^2.5.1",
-    "turbo": "^1.1.3"
+    "@babel/core": "^7.17.10",
+    "@vercel/style-guide": "^3.0.0",
+    "cross-env": "^7.0.3",
+    "eslint": "^8.12.0",
+    "husky": "^7.0.4",
+    "prettier": "^2.6.1",
+    "turbo": "^1.2.6",
+    "typescript": "^4.6.4"
   },
   "engines": {
-    "npm": ">=7.0.0",
-    "node": ">=12.0.0"
+    "node": ">=14.0.0"
```

```suggestion
    "node": ">=12.0.0"
```

Does this really require 14 or newer? We should typically not drop support unless we're doing semver major release.